portfolio/src/main/webapp/script-map.js

```diff
@@ -0,0 +1,144 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+let map;
+const flagIcon = 'https://developers.google.com/maps/documentation/javascript/examples/full/images/beachflag.png';
+
+/* Editable marker that displays when a user clicks in the map. */
+let editMarker;
+
+/** Creates a map that shows travel markers and allows user to input markers. */
+function createMap() {
+  map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 39.329858, lng: -76.620540}, zoom: 5});
```

Comment:
I would define all these LatLngs  as constants.
Something like a map from a string -> array where the array has {lat, lng}.

Then you can access them using the map.

Reply:
Or even better, for all markers  create an Object that will contain the name, description, lat, lng. 
Then you can access it it with variableName.lat, variableName.lng, etc.

this is for lines 36 onwards

Reply:
I'm not sure what you mean by creating a LatLngs map that is constant? That would need me to change the values of LatLngs.lat and LatLngs.lng every time when I try to create a marker position, no?

Reply:
Also about creating an Object, I think Marker is already an Object in the library and it is created by `new google.maps.Marker({position: {lat: lat, lng: lng}, map: map});`